st2actions/st2actions/handlers/mistral.py

```diff
@@ -62,22 +62,32 @@ def callback(url, context, status, result):
                 # tracker before sending the output.
                 result.pop('tasks', None)
 
+            match = re.search('(.*)/action_executions/(.*)', url)
```

Comment:
Need to use `+` (1 or more) here instead of `*` (zero or more) otherwise, there will always be 2 matches if the URL contains at least the `/action_executions/` string.

It also might be worth splitting this into a method so it can be tested.